includes/rcc.h

```diff
@@ -39,14 +36,15 @@ typedef enum    UART_PORT_e
 {
     UART_PORT_1 = 14,
     UART_PORT_2 = 17,
-    UART_PORT_3,
-    UART_PORT_4,
-    UART_PORT_5,
+    UART_PORT_3 = 18,
+    UART_PORT_4 = 19,
+    UART_PORT_5 = 20,
     UART_PORT_6 = 5,
-    UART_PORT_7,
-    UART_PORT_8
+    UART_PORT_7 = 6,
+    UART_PORT_8 = 7
 }               UART_PORT;
 
+void    rcc_init_clocks();
```

Comment:
```suggestion
void    rcc_init_clocks(void);
```

Otherwise, it's a vararg function in C.